<launch>
    <include file="$(find avoidance)/launch/local_planner_sitl_mavros.launch" >
        <arg name="model" value="iris_depth_camera" />
        <arg name="world" value="outdoor_village_3" />
        <arg name="point_cloud_topic" value="/camera/depth/points" />
    </include>

     <node name="local_planner_node" pkg="avoidance" type="local_planner_node" >
        <param name="goal_x_param" value="46" />
        <param name="goal_y_param" value="42"/>
        <param name="goal_z_param" value="8" />
    </node>

    <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find avoidance)/resource/local_planning.rviz" />
</launch>
